File: php/sendsms.php
<?php


function sendSms($phoneNumber,$message){
    $apiKey="YOUR API_KEY";
    $sendeName="YOUR ASSIGNED SENDER NAME";

    $bodyRequest =array(
        "mobile" =>$phoneNumber,
        "response_type"=>"json",
        "sender_name"=>$sendeName,
        "service_id"=>0,
        "message"=>$message
    );
    $payload = json_encode($bodyRequest);
    $curl = curl_init();
    curl_setopt_array($curl, array(
    CURLOPT_URL => 'https://api.mobitechtechnologies.com/sms/sendsms',
    CURLOPT_RETURNTRANSFER => true,
    CURLOPT_ENCODING => '',
    CURLOPT_MAXREDIRS => 10,
    CURLOPT_TIMEOUT => 15,
    CURLOPT_FOLLOWLOCATION => true,
    CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
    CURLOPT_CUSTOMREQUEST => 'POST',
    CURLOPT_POSTFIELDS =>$payload,
    CURLOPT_HTTPHEADER => array(
        'h_api_key: '.$apiKey,
        'Content-Type: application/json'
    ),
    ));

    $response = curl_exec($curl);
    curl_close($curl);
    echo $response;

}

sendSms("2547XXXXXXXX",
"This is a message.\n\nRegards\nMobiTech Technologies");

?>

File: python/sendsms.py
import requests
import json

url = "https://api.mobitechtechnologies.com/sms/sendsms"

# Sample JSON data
data = {
    "mobile": "254702XXXXX",
    "response_type": "json",
    "sender_name": "23107",
    "service_id": 0,
    "message": "This is a message.\n\nRegards\nMobiTech Technologies"
}

# Convert data to JSON format
json_data = json.dumps(data)

# Set the content type header
headers = {
    'Content-type': 'application/json',
    'h_api_key':'YOUR_API_KEY'}

# Make the request
response = requests.post(url, data=json_data, headers=headers)

# Check the response status code
if response.status_code == requests.codes.ok:
    # Parse the JSON data
    parsed_response = json.loads(response.content)

    # Access the data
    for item in parsed_response:
        print(f"Status code:", item["status_code"])
        print(f"Status description:", item["status_desc"])
        print(f"Message ID:", item["message_id"])
        print(f"Mobile number:", item["mobile_number"])
        print(f"Network ID:", item["network_id"])
        print(f"Message cost:", item["message_cost"])
        print(f"Credit balance:", item["credit_balance"])
else:
    print("Request failed")
